src/handlers/GraphQLHandler.ts

```diff
@@ -108,7 +109,10 @@ export class GraphQLHandler<
   }
 
   parse(request: MockedRequest) {
-    return parseGraphQLRequest(request)
+    return tryCatch(
+      () => parseGraphQLRequest(request),
+      (error) => console.error(error.message),
```

Comment:
Handling parsing exceptions as console errors to preserve the previous behavior. At the moment, when parsing a request in a _graphql handler_ fails, it doesn't produce a 500 response, but a console error including the parsing error.